test/common/services/security/test_user.ts

```diff
@@ -52,6 +52,10 @@ export class TestUser extends FtrService {
       return;
     }
 
+    if (process.env.FTR_ENABLE_FIPS_AGENT?.toLowerCase() === 'true') {
```

Comment:
thoughts on adding a debug log here?  for my understanding, why are we returning early?

Reply:
++ I will definitely add a log statement here

I was able to force more tests to pass by not allowing roles to be overridden; some tests (when security was enabled) start using Test User when they were expecting no authc/authz so I set the defaultRoles to the most privileged roles I could. 

here is a test CI run with that line disabled: https://github.com/elastic/kibana/pull/192088

Screenshot of what I was trying to work around:
<img width="1583" alt="Screenshot 2024-09-04 at 11 05 13 AM" src="https://github.com/user-attachments/assets/d5641bf1-009d-46fc-b2c6-2677d4f7046e">